src/main/java/core/basesyntax/exceptions/ExpectedException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class ExpectedException extends Exception {
```

Comment:
1) name of exception should represent case when it's used
2) please use extends RuntimeException in order to avoid interface signature changes
```suggestion
public class RegistrationException extends RuntimeException {
```